stm32_base/hal/usart_stm32/usart_device_stm32.c

```diff
@@ -699,6 +699,16 @@ void USART1_IRQHandler(void)
         xQueueSendFromISR(xUsart0Rx, &cChar, &xTaskWokenByPost);
     }
 
+    if (USART_GetITStatus(USART1, USART_IT_ORE_RX) != RESET) {
+        /* Handle Overrun error
+         * This bit is set by hardware when the word currently being received in the shift register is
+           ready to be transferred into the RDR register while RXNE=1. An interrupt is generated if
+           RXNEIE=1 in the USART_CR1 register. It is cleared by a software sequence (an read to the
+           USART_SR register followed by a read to the USART_DR register) */
```

Comment:
So... I'm confused.  This gets set every time that RXNE is set?  Or is it set when we don't read a character out of the receive buffer fast enough and the buffer overruns?